bind9/9.11.22.patch

```diff
@@ -0,0 +1,937 @@
++	AC_DEFINE([HAVE_EVP_CIPHER_CTX_FREE], [1], [wolfSSL includes])
++	AC_DEFINE([HAVE_EVP_CIPHER_CTX_NEW], [1], [wolfSSL includes])
++	AC_DEFINE([HAVE_EVP_MD_CTX_RESET], [1], [wolfSSL includes])
++	AC_DEFINE([HAVE_EVP_MD_CTX_FREE], [1], [wolfSSL includes])
++	AC_DEFINE([HAVE_EVP_MD_CTX_NEW], [1], [wolfSSL includes])
++	AC_DEFINE([HAVE_EVP_SHA1], [1], [wolfSSL includes])
++	AC_DEFINE([HAVE_EVP_SHA224], [1], [wolfSSL includes])
++	AC_DEFINE([HAVE_EVP_SHA256], [1], [wolfSSL includes])
++	AC_DEFINE([HAVE_EVP_SHA384], [1], [wolfSSL includes])
++	AC_DEFINE([HAVE_EVP_SHA512], [1], [wolfSSL includes])
++	AC_DEFINE([HAVE_RSA_SET0_KEY], [1], [wolfSSL includes])
++	AC_DEFINE([HAVE_TLS_SERVER_METHOD], [1], [wolfSSL includes])
++	AC_DEFINE([HAVE_HMAC_CTX_NEW], [1], [wolfSSL includes])
++	AC_DEFINE([HAVE_HMAC_CTX_FREE], [1], [wolfSSL includes])
++	AC_DEFINE([HAVE_HMAC_CTX_RESET], [1], [wolfSSL includes])
++	AC_DEFINE([HAVE_HMAC_CTX_GET_MD], [1], [wolfSSL includes])
++	AC_DEFINE([HAVE_DH_GET0_KEY], [1], [wolfSSL includes])
++	AC_DEFINE([HAVE_DSA_GET0_PQG], [1], [wolfSSL includes])
++	AC_DEFINE([HAVE_OPENSSL_EVP_AES], [1], [wolfSSL includes])
++	AC_DEFINE([HAVE_OPENSSL_AES], [1], [wolfSSL includes])
++	use_openssl="wolfssl"
++fi
+ 
+ #
+ # was --with-pkcs11 specified?
+@@ -1587,6 +1651,25 @@ case "$use_openssl" in
+ [OpenSSL was not found in any of $openssldirs; use --with-openssl=/path
+ If you don't want OpenSSL, use --without-openssl])
+ 		;;
++	wolfssl)
++		AC_MSG_RESULT(Using wolfSSL)
++		DST_OPENSSL_INC="${WOLFSSL_CFLAGS}"
++		DST_OPENSSL_LIBS="${WOLFSSL_LIBS}"
++		CRYPTO="-DHAVE_WOLFSSL -DOPENSSL"
++		OPENSSL_ECDSA="yes"
++		OPENSSLECDSALINKOBJS='${OPENSSLECDSALINKOBJS}'
++		OPENSSLECDSALINKSRCS='${OPENSSLECDSALINKSRCS}'
++		AC_DEFINE(HAVE_OPENSSL_ECDSA, 1,
++			  [Define if your OpenSSL version supports ECDSA.])
++		OPENSSLEDDSALINKOBJS=""
++		OPENSSLEDDSALINKSRCS=""
++		OPENSSLGOSTLINKOBJS=""
++		OPENSSLGOSTLINKSRCS=""
++		OPENSSLLINKOBJS='${OPENSSLLINKOBJS}'
++		OPENSSLLINKSRCS='${OPENSSLLINKSRCS}'
++		ISC_OPENSSL_INC="$DST_OPENSSL_INC"
++		ISC_OPENSSL_LIBS="$DST_OPENSSL_LIBS"
++		;;
+ 	*)
+ 		if test "yes" = "$want_native_pkcs11"
+ 		then
+@@ -5508,8 +5591,13 @@ report() {
+     if test "yes" = "$enable_full_report"; then
+ 	test "no" = "$enable_ipv6" -o "no" = "$found_ipv6" || \
+ 	    echo "    IPv6 support (--enable-ipv6)"
+-	test "X$CRYPTO" = "X" -o "yes" = "$want_native_pkcs11" || \
+-		echo "    OpenSSL cryptography/DNSSEC (--with-openssl)"
++	if test "no" != "$use_wolfssl"
++	then
++		echo "    wolfSSL cryptography (--with-wolfssl)"
++	else
++		test "X$CRYPTO" = "X" -o "yes" = "$want_native_pkcs11" || \
++			echo "    OpenSSL cryptography/DNSSEC (--with-openssl)"
++	fi
+ 	test "X$PYTHON" = "X" || echo "    Python tools (--with-python)"
+ 	test "X$XMLSTATS" = "X" || echo "    XML statistics (--with-libxml2)"
+ 	test "X$JSONSTATS" = "X" || echo "    JSON statistics (--with-libjson)"
+diff --git a/doc/dev/unexpected b/doc/dev/unexpected
```

Comment:
It looks like this file doesn't exist in 9.11.22.

Reply:
Are you using the gitlab source or a release bundle?

Reply:
Release bundle via https://downloads.isc.org/isc/bind9/9.11.22/. I can see how the customer is pulling down the source in their Yocto environment, but my guess is release bundle. Will post an update shortly.

Reply:
The Yocto build is pulling the source code from https://ftp.isc.org/isc/bind9/9.11.22/bind-9.11.22.tar.gz, so ideally the patch should apply cleanly onto that, rather than the git repo.

Reply:
Does the missing file make the patch fail? Or does it just issue a warning?

Reply:
It fails. It causes an error and kills the build at the `do_patch` stage in the Yocto environment.

Reply:
File removed.